### PR TITLE
Multicast feature deployment

### PR DESCRIPTION
**Pull request by BiliouriV**
# PR Checklist 

### _1- PR Comments_
    *If needed, to be filled by developer*  

<br><hr/>
*WARNING : Do not edit the checklist below.*  
<hr/><br>

### _2- Developer section_

- [x] [Documentation] is up to date with new feature
- [x] [Tests] are *passed OK* (Non regression, new features & bug fixes)
- [x] [Code Quality] Remind to check if :
    * Each function has a header (description, inputs, outputs) 
    * Code is commented (particularly in hard to understand areas)
    * There are no new warnings that can be corrected
    * Commits policy is respected (constitancy commits, clear commits comments)
<hr/><br>

### _3- QA section_

- [x] [Review] tests for new features have been *reviewed*
- [x] [Changelog] is up-to-date with expected tags  
    🆕 Feature: [Feature] Description...  
    🆕 Added: [Feature] Description...  
    🆕 Changed: [Feature] Description...  
    🛠️ Fix: [Feature] Description...

### CODE REVIEW

File: network/robus/inc/config.h [hunk 1]
```diff
@@ -45,8 +45,8 @@
 #endif
 
 // Tab of byte. + 2 for overlap ID because aligned to byte
-#define MASK_SIZE ((MAX_SERVICE_NUMBER / 8) + 2)
-
+#define MASK_SIZE           ((MAX_SERVICE_NUMBER / 8) + 2)
```
Comment (Salem-Luos) [inline network/robus/inc/config.h:48]: ID_MASK_SIZE instead of MASK_SIZE

File: network/robus/src/robus.c [hunk 2]
```diff
@@ -75,6 +76,15 @@ void Robus_Init(memory_stats_t *memory_stats)
     baudrate = DEFAULTBAUDRATE;
     // mask
     Robus_MaskInit();
+
+    // multicast mask init
+    ctx.ShiftMulticastMask = 0;
+    ctx.MulticastTargetMax = 0;
```
Comment (Salem-Luos) [inline network/robus/src/robus.c:82]: Global variable name MulticastTargetNumer

Comment (nicolas-rabault) [reply network/robus/src/robus.c:82]: Agree, if you don't need those variables on other files you should put them as file global variables.

**Comment by BiliouriV** [comment]
@nicolas-rabault I agree that we should change the target_mode = ID to SERVICE but when we will do the major Luos clean and not now. If you want create an issue regarding this or I can do it

File: network/robus/inc/config.h (hunk 1 above)
Comment (BiliouriV) [reply network/robus/inc/config.h]: @nicolas-rabault I agree that we should change the target_mode = ID to SERVICE but when we will do the major Luos clean and not now. If you want create an issue regarding this or I can do it

Comment (nicolas-rabault) [reply network/robus/inc/config.h]: Agree, yes please feel free to create an issue about it.

File: network/robus/src/reception.c [hunk 3]
```diff
@@ -387,23 +387,44 @@ ll_service_t *Recep_GetConcernedLLService(header_t *header)
...
+ * @param target of message
+ * @return None
+ ******************************************************************************/
+static inline error_return_t Recep_MulticastCompare(uint16_t target)
+{
+    uint8_t compare = 0;
+    // make sure there is a topic that can be received by the node
+    if (target <= LAST_TOPIC)
+    {
+        digitalWrite(9, true);
```
Comment (nicolas-rabault) [inline network/robus/src/reception.c:418]: I think this line should not be here...

**Review by nicolas-rabault — CHANGES_REQUESTED** [review]
There are some debugging lines remaining...

File: network/robus/src/reception.c (hunk 3 above)
Comment (BiliouriV) [reply network/robus/src/reception.c:418]: yes

**Review by Salem-Luos — APPROVED** [review]
ok for me!!!! good job!

**Review by JeromeGalan — APPROVED** [review]

**Pull request merged**